src/Gui/DlgToolbarsImp.cpp

```diff
@@ -32,6 +32,7 @@
 #endif
 

```

Comment:
The changes on this class cause some regressions:

 * Whenever opening the customize dialog and switching to the Toolbars tab it only shows the "Separator" item on the left side. In order to fill up this list one has to manually select a category, first.

* When the list is filled the "Separator" item is gone.